file-formats/evtb/README.md

```diff
@@ -0,0 +1,6 @@
+# EVTB File Format
+This file format is used for the ADT Editor for EVTB
```

Comment:
This file format is used by the ADT editor for RAP Business Event Bindings (EVTB).